### PR TITLE
Updating windows-runtimeclass-support KEP for 1.19

### PR DESCRIPTION
**Pull request by marosset**

### CODE REVIEW

File: keps/sig-windows/windows-runtimeclass-support.md [hunk 1]
```diff
@@ -447,8 +462,9 @@ In addition to what's included there, for beta:
 
 #### Beta -> GA Graduation
 
-- Define & add E2E testing for recommended default RuntimeClass configurations for Windows supported with CRI-ContainerD
+> Proposed for 1.21 or later
```
Comment (benmoss) [inline keps/sig-windows/windows-runtimeclass-support.md:465]: I'd just leave these tentative versions out completely

Comment (marosset) [reply keps/sig-windows/windows-runtimeclass-support.md:465]: Yea... let me remove these

**Comment by benmoss** [comment]
Whoops maybe should have held for others to approve

**Comment by michmike** [comment]
its fine Ben! LGTM too

**Comment by marosset** [comment]
@benmoss I'll update with any additional feedback

**Pull request merged**